routes/users.js

```diff
@@ -17,6 +17,7 @@ router.patch("/self", authenticate, userValidator.updateUser, users.updateSelf);
 router.get("/", userValidator.getUsers, users.getUsers);
 router.get("/self", authenticate, users.getSelfDetails);
 router.get("/isUsernameAvailable/:username", authenticate, users.getUsernameAvailabilty);
```

Comment:
The API name can be corrected to `/users/username/:username` to better

Reply:
sorry for late response. My health is not good last week. I am OOO

Regarding your comment, This API route will get removed